### ISSUE TITLE
Picking the correct window size

### ISSUE THREAD

**Issue by khufkens** [bluegreen-labs/foto#8]
The correct window size for your analysis is dependent on:

- the texture(s) of your data (crown sizes)
- the resolution of you data (relative to the true crown sizes)
- the coverage desired

When selecting your window size you want to make sure that the coarsest texture is captured by such a window. A window size below this coarsest texture in the analysis will lead to poorly classified textures (across the whole spectrum of your dataset). Including too much data, with a large window, will equally decrease the ability of the algorithm to capture all textures well.

In short, the choice of the window depends on the input data and texture types encountered.

To increase the coverage, spatially of the classification output you can use a moving window approach. Rather than using zones, and decreasing the resolution, by a factor scaling with the window size, every pixel and its surrounding will be considered. The resolution of the input data is therefore maintained.

This topic was brought to my attention by @vincent-haller

**Comment by khufkens** [comment]
A preliminary vignette is here:

https://bluegreen-labs.github.io/foto/articles/foto-background.html

This documents the basic theoretical background.
